.circleci/config.yml

```diff
@@ -37,6 +37,46 @@ commands:
             - run: sudo apt-get install ocl-icd-opencl-dev libhwloc-dev
       - run: git submodule sync
       - run: git submodule update --init
+  download-params:
+    steps:
+      - restore_cache:
+          name: Restore parameters cache
+          keys:
+            - 'v25-2k-lotus-params'
+          paths:
+            - /var/tmp/filecoin-proof-parameters/
+      - run:  lotus fetch-params 2048
+      - save_cache:
+          name: Save parameters cache
+          key: 'v25-2k-lotus-params'
+          paths:
+            - /var/tmp/filecoin-proof-parameters/
+  lotus-artefacts:
+    steps:
+      - restore_cache:
+          name: Restore lotus artefacts cache
+          keys:
+            - 'lotus-v1.13.1-rc.22'
+          paths:
+            - /tmp/lotus-artefacts
+      - setup_remote_docker:
+          version: 19.03.13
+      - run:
+          name: copy latest stable lotus binaries
+          command: |
+            docker pull nonsens3/lotus:v1.13.1-rc2
```

Comment:
We need docs on this, given that there is no existing image we can leverage... it will be up to us to update the version of Lotus periodically.